src/libs/PopUpNotification.js

```diff
@@ -0,0 +1,138 @@
+import React, {
+    forwardRef, useImperativeHandle, useRef, useState,
+} from 'react';
+import {
+    StyleSheet, Text, View, Animated, Platform,
+} from 'react-native';
+import {
+    Directions, FlingGestureHandler, State, TouchableWithoutFeedback,
+} from 'react-native-gesture-handler';
+import colors from '../styles/colors';
+import Icon from '../components/Icon';
+import {Checkmark, Exclamation} from '../components/Icon/Expensicons';
+import ScreenWrapper from '../components/ScreenWrapper';
+import styles from '../styles/styles';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../components/withWindowDimensions';
+
+const desktopContainerStyle = {
+    maxWidth: '380px',
+    top: '20px',
+    right: 0,
+    position: 'fixed',
+};
+
+const popupStyles = StyleSheet.create({
```

Comment:
@roryabraham I forget where we landed here, but do we want these styles living here in the component JS file or should they all be in styles.js?

Reply:
They should all be in `styles.js`. We wrote up some documentation around our RN style guidelines [here](https://github.com/Expensify/Expensify.cash/blob/main/STYLING.md)

Reply:
I moved all styles to `styles.js` in my last commit